gossip/evmstore/store.go

```diff
@@ -126,6 +126,31 @@ func (s *Store) RebuildEvmSnapshot(root common.Hash) {
 	s.Snaps.Rebuild(root)
 }
 
+// Commit changes.
+func (s *Store) CleanCommit(block iblockproc.BlockState) error {
```

Comment:
We may change it so that `CleanCommit` would get executed inside `evmstore.Store.Commit` if `flush=true`. It seems that we should either have 2 functions (`CleanCommit` and `Commit`) but without the `flush` argument, or only `Commit` function with the `flush` argument

Reply:
But with the current code if `flush=true` mean we does not clean any things but commit everything to the db like archive mode